docs/docs/howto/authentication/google-account.rst

```diff
@@ -0,0 +1,54 @@
+Google Account Login
+====================
+
+Users can login with their Google account credentials. On the login screen user, is prompted with a login option for Google account.
+
+After clicking on the login button, user is provided with an Google account pop up window with options to provide his/her credentials or select a Google account.
+
+* Configuration
+
```

Comment:
Same comment here & also for third party etc entries below.

Reply:
@aindriu-aiven pushed new changes